.travis.yml

```diff
@@ -112,6 +113,8 @@ script:
   - python -c 'import numpy as np; print("relaxed strides checking:", np.ones((10,1),order="C").flags.f_contiguous)'
   # Make sure that relaxed strides checking is actually in effect; otherwise fail loudly
   - if [ "$NPY_RELAXED_STRIDES_CHECKING" == "1" ]; then python -c'import numpy as np; assert np.ones((10,1),order="C").flags.f_contiguous'; fi
+  # Test that mpmath actually uses gmpy2
+  - python -c 'import mpmath.libmp; assert mpmath.libmp.BACKEND == "gmpy"'
```

Comment:
> No need for a separate issue, I'd think.

I mean that build script can open a new issue (or reopen some specific old one) in case of failure. This all is possible, but seems too complex for me.